src/test/java/org/killbill/billing/plugin/bridge/TestBridgeConfigConfigurationHandlerForTest.java

```diff
@@ -1,40 +0,0 @@
-/*
- * Copyright 2014-2018 Groupon, Inc
- * Copyright 2014-2018 The Billing Project, LLC
- *
- * The Billing Project licenses this file to you under the Apache License, version 2.0
- * (the "License"); you may not use this file except in compliance with the
- * License.  You may obtain a copy of the License at:
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
- * License for the specific language governing permissions and limitations
- * under the License.
- */
-
-package org.killbill.billing.plugin.bridge;
-
-import java.util.UUID;
-
-import org.testng.Assert;
-import org.testng.annotations.Test;
-
-public class TestBridgeConfigConfigurationHandlerForTest {
-
-    @Test(groups = "fast")
-    public void testParseYAML() {
-        final BridgeConfigConfigurationHandlerForTest bridgeConfigConfigurationHandlerForTest = new BridgeConfigConfigurationHandlerForTest("eu-central-1");
-        final BridgeConfig bridgeConfig = bridgeConfigConfigurationHandlerForTest.getConfigurable(UUID.randomUUID());
-
-        Assert.assertEquals(bridgeConfig.killbillClientConfig.serverUrl.toString().compareTo("http://kb.eu:8080"), 0);
-        Assert.assertNull(bridgeConfig.paymentConfig.getControlPlugins());
-        Assert.assertEquals(bridgeConfig.paymentConfig.getPluginProperties().size(), 2);
-        Assert.assertEquals(bridgeConfig.paymentConfig.getPluginProperties().get(0).getKey(), "skipGw");
-        Assert.assertEquals(bridgeConfig.paymentConfig.getPluginProperties().get(0).getValue(), true);
-        Assert.assertEquals(bridgeConfig.paymentConfig.getPluginProperties().get(1).getKey(), "deposit");
-        Assert.assertEquals(bridgeConfig.paymentConfig.getPluginProperties().get(1).getValue(), "{\"system\":[{\"type\":\"default\",\"corporation\":\"acme\"}]}");
```

Comment:
Why did you remove it? It was in place to verify the serialization of JSON properties.

Reply:
Mistake will fix.